src/ObjCRuntime/Blocks.cs

```diff
@@ -341,7 +341,7 @@ internal static IntPtr GetBlockForDelegate (MethodInfo minfo, object @delegate,
 		}
 
 		[DllImport (Messaging.LIBOBJC_DYLIB)]
-		static extern IntPtr _Block_copy (ref BlockLiteral block);
+		internal static extern IntPtr _Block_copy (ref BlockLiteral block);
```

Comment:
needed ? it's already (re)declared in `DispatchBlock` ?

Reply:
I've moved all the P/Invokes in DispatchBlock here.